pgpwordlist/pgpwordlist.go

```diff
@@ -35,7 +35,7 @@ func ByteToMnemonic(b byte, index int) string {
 func DecodeMnemonics(words []string) ([]byte, error) {
 	decoded := make([]byte, len(words))
 	idx := 0
-	for _, w := range words[:len(words)] {
+	for _, w := range words[:] {
```

Comment:
even simpler would be `for _, w := range words {` since it's already a slice. weird it didn't simplify this all the way.